data/sql/updates/pending_db_world/rev_1721586059640959700.sql

```diff
@@ -0,0 +1,9 @@
+--
+DELETE FROM `command` WHERE `name`='aura stack';
+INSERT INTO `command` VALUES ('aura stack', 2, 'Syntax: .aurastack #spellid #stacks\r\n\r\nModify #stacks of an already applied #spellid to the selected Unit.');
+
+DELETE FROM `acore_string` WHERE `entry` IN (373,374,375);
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
+(373, 'Target unit does not have aura %u!'),
+(374, 'No stack amount specified!'),
+(375, 'Spell %u cannot have stacks!');
```

Comment:
```suggestion
INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
(373, 'Target unit does not have aura {}!'),
(374, 'No stack amount specified!'),
(375, 'Spell {} cannot have stacks!');
```

Reply:
not a single string in acore_string is using curly brackets.

Reply:
wrong

Reply:
https://github.com/azerothcore/azerothcore-wotlk/commit/3fbded8f6bdaae1ec6897dae34afdab36b124899

Reply:
sneaky